snapcraft/application.py

```diff
@@ -98,14 +98,14 @@ def __init__(self, *args, **kwargs) -> None:
         # compatibility with previous versions of the snapcraft codebase, and in
         # the package service to copy the project file into the snap payload if
         # manifest generation is enabled.
-        self._known_core24 = self._get_known_core24()
+        self._use_craftapp_lib = self._should_use_craftapp_lib()
 
         for craft_var, snapcraft_var in MAPPED_ENV_VARS.items():
             if env_val := os.getenv(snapcraft_var):
                 os.environ[craft_var] = env_val
 
-    def _get_known_core24(self) -> bool:
-        """Return true if the project is known to be core24."""
+    def _should_use_craftapp_lib(self) -> bool:
+        """Return true if the project is known to be core24 or newer."""
```

Comment:
```suggestion
        """Return true if the project is known to use Craft Application to build."""
```
It's right either way, but I figure the more generic description is more useful